distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -150,6 +150,7 @@ ALERT;Transformations-MAP: "-" entry defined in a MAP file is no more used by si
 [4.3.0]
 ALERT;JavaScript Automation: The isJsInstanceOfJavaType method of the utils namespace has been removed. Use JavaScript's instanceof operator instead.
 ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been organized into groups. You will need to reconfigure your setup to the new thing structure.
+ALERT;ElectroluxAir binding: The binding has been removed since the Electrolux Delta API has been discontinued.
```

Comment:
```suggestion
ALERT;ElectroluxAir Binding: The binding has been removed since the Electrolux Delta API has been discontinued.
```

Can you also keep the list sorted by moving it before the JavaScript Automation ALERT? :slightly_smiling_face: